docs/content/composer/README.md

```diff
@@ -1,40 +1,62 @@
 ---
 order: 0
+headerDepth: 0
 ---
 
-# Composer
+# About Composer
 
-Composer is a free, open-source, multiplayer, local-first knowledge environment powered by DXOS. Users can collaborate real-time, work offline, and retain privacy through the peer-to-peer protocols of DXOS. User data is never stored on any servers and does not leave devices the user trusts. Composer works entirely on the client with no private data silos.
+*Composer is built on the DXOS framework. If you want to know more about the framework, check out the [SDK docs](../guide).*
 
-Developers can extend Composer through plugins - every feature is implemented as a plugin that can be swapped or extended. Community contributions are most welcome.
+Composer is a team notetaking, whiteboarding, and knowledge management program.
 
-[Try Composer](https://composer.dxos.org){class="button"}
+It has three key features:
 
-<a id="technology-preview"></a>
-::: warning Technology Preview
-Composer is not yet ready for production. There are bugs, and breaking changes may occur. Migration guides will be provided for version updates in the [release notes](https://github.com/dxos/dxos/releases).
+* **Extensibility**: If your team is operating in a challenging problem domain you need knowledge tools specific to your work. Composer is built for customization from the ground up.
+* **Collaboration**: Composer has real-time multiplayer implemented through a local-first model. This makes a lot of things better: from meeting notes to working on airplanes.
+* **Openness**: Not only are Composer and the DXOS protocols it's built on open source, the application is cloudless as well. Due to the [CRDT](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type)-based implementation there's no need for a server other than for backup. DXOS provides backup services if you want them, but you can also host your own.
 
-All your feedback is most welcome on [Discord](https://discord.gg/eXVfryv3sW).
-:::
+## Peer-to-peer
+
+Composer gets its real-time collaboration and offline abilities through a peer-to-peer model.
+
+For example a running system might sync data like this, where nodes represent devices and edges represent syncing connections:
 
-## Key Features
+```mermaid
```

Comment:
I'm making a nicer Figma graph for illustration next, but I'd rather not block this PR for it. We can either use the current not-great mermaid diagram, or maybe it would be better to just delete this section for now.